testCourse/questions/demo/imageCapture/question.html

```diff
@@ -0,0 +1,17 @@
+<pl-question-panel>
+    <p>Consider the function $f(x) = x^3+2x^2+x+1$. Draw the graphs of the following functions: </p>
```

Comment:
Nit, consistent indentation:

```suggestion
  <p>Consider the function $f(x) = x^3+2x^2+x+1$. Draw the graphs of the following functions: </p>
```